tests/test_collection.py

```diff
@@ -314,8 +312,8 @@ def test_collection_mapped():
     assert len(ls_ds[0]) == 2 and len(ls_ds[2]) == 2
 
     # adata3 goes first here
-    with collection_outer.mapped(label_keys="feat1", join_vars="auto") as ls_ds:
-        assert ls_ds.join_vars == "outer"
+    with collection_outer.mapped(label_keys="feat1", join="inner") as ls_ds:
+        assert ls_ds.join_vars == "inner"
```

Comment:
collection_outer has adatas with non-intersecting vars, this collection should be used to test outer join, not inner.

Reply:
Fixed!